schema.graphql

```diff
@@ -299,6 +303,8 @@ type Subgraph @entity {
   signalledTokens: BigInt!
   "CUMULATIVE unsignalled tokens on this subgraph all time"
   unsignalledTokens: BigInt!
+  "CURRENT amount of tokens signalled on this subgraph. Only accounts for signalling done through GNS (name signalling)"
+  currentSignalledTokens: BigInt!
```

Comment:
Good point, the main idea is that the frontend needs to have a way of displaying the current amount of GRT signalled for a subgraph, and also have that value be sortable (so it needs to be a specific field, and it needs to be on the `Subgraph` entity).

Since the `Subgraph` entity representation of signalling is `NameSignal` (given the long discussion we had about signal vs nameSignal, double counting curation and what is a subgraph), and given it had 2 cumulative counters (`signalledTokens` and `unsignalledTokens`) I basically replicated the `signalledTokens - unsignalledTokens` value on a new separate field, which should ideally be the amount of GRT signalled through name signals, but now that you mention it, unsignalled can be higher if people curate outside of the GNS name signal system, and someone from the GNS name signal system unsignals...

All I can think as a better solution is to directly link the latest `SubgraphDeployment` entity, and have this new field track the token changes with version signals instead 🤔, or something along those lines

Reply:
Gonna leave this implemented like this, so it will still have this "known issue" where it can be negative on full unsignals.

It's at least better than have it be a cumulative value used as a current value, and the other alternatives will require something like we did for delegators in the analytics subgraph.